polls/views.py

```diff
@@ -286,6 +287,51 @@ def test_func(self):
         return self.request.user.is_superuser
 
 
+class AdminBatchSubmitView(LoginRequiredMixin, UserPassesTestMixin, View):
+    def get(self, request):
+        experiments = Experiment.objects.all()
+        context = {"experiments": experiments}
+        return render(request, "polls/admin-batch-submit.html", context)
+
+    def post(self, request, *args, **kwargs):
+        json_data = request.POST.get("json-data")
+        exp_pk = request.POST.get("exp_pk")
+        experiment_id = Experiment.objects.filter(pk=int(exp_pk))
+        db_data = json.loads(json_data)
+        if experiment_id.exists():
+            experiment = Experiment.objects.get(pk=exp_pk)
+            new_batch = Batch.objects.create(
+                is_gold=db_data["is_gold"] if db_data["is_gold"] else False,
+                notes=db_data["notes"] if db_data["notes"] else "",
+                experiment=experiment,
+            )
+            new_batch.save()
+            for task in db_data["tasks"]:
+                if "transform_metadata" in task:
+                    new_task = Task.objects.create(
+                        batch=new_batch,
+                        reference_url=task["reference_url"],
+                        transform_url=task["transform_url"],
+                        transform_metadata=task["transform_metadata"],
+                    )
+                    new_task.save()
+                else:
```

Comment:
Do we really need an else here? we can just put transform_metadata as "None" and create new Task object. This will help n making the code leaner and simpler